share/cmake/modules/Findyaml-cpp.cmake

```diff
@@ -41,32 +40,39 @@ if(CMAKE_BUILD_TYPE MATCHES "[Dd][Ee][Bb][Uu][Gg]")
    set(BUILD_TYPE_DEBUG ON)
 endif()
 
+if(yaml-cpp_FIND_QUIETLY)
+    set(quiet QUIET)
+endif()
+
 if(NOT OCIO_INSTALL_EXT_PACKAGES STREQUAL ALL)
-    set(_yaml-cpp_REQUIRED_VARS yaml-cpp_LIBRARY)
 
+    # Search for yaml-cpp-config.cmake
     if(NOT DEFINED yaml-cpp_ROOT)
-        # Search for yaml-cpp-config.cmake
-        find_package(yaml-cpp ${yaml-cpp_FIND_VERSION} CONFIG QUIET)
+        find_package(yaml-cpp ${yaml-cpp_FIND_VERSION} CONFIG ${quiet})
     endif()
 
     if(yaml-cpp_FOUND)
-        get_target_property(yaml-cpp_LIBRARY yaml-cpp LOCATION)
+        # Alias target for yaml-cpp < 0.8 compatibility
+        if(TARGET yaml-cpp AND NOT TARGET yaml-cpp::yaml-cpp)
+            add_library(yaml-cpp::yaml-cpp ALIAS yaml-cpp)
+        endif()
+
+        set(yaml-cpp_INCLUDE_DIR ${YAML_CPP_INCLUDE_DIR})
```

Comment:
[Minor] Don't think this is needed here?

Reply:
The module documents that this value will be set on line 9. I would prefer to keep that consistent.

Reply:
Fair enough, this duplicated set of exported variables (upper / lower case) is a bit confusing, might be good to remove at a later point.